### ISSUE TITLE
Player lost their money and their homes !

### ISSUE THREAD

**Issue by Proemmis** [Zrips/CMI#9098]
### Description of Issue

Hi, after update to newer version, playerslost their homes and their money.

My cmi.sqlite.db file is not touched 
![image](https://github.com/user-attachments/assets/0db86cd4-a9a3-470c-859e-856e9855de78)

### Version Information

```shell
cmi version
[16:55:59] [Server thread/INFO]: --------------------------------------------------
[16:55:59] [Server thread/INFO]: CMI: 9.7.5.6 SqLite-> 9.7.5.5
[16:55:59] [Server thread/INFO]: CMILib: 1.5.1.2
[16:55:59] [Server thread/INFO]: Serveur: Spigot 1.12.2-R0.1-SNAPSHOT-
[16:55:59] [Server thread/INFO]: CMI economy: Activé Vault: 1.7.3-b131(CMIEInjector) CMI Chat: Activé
[16:55:59] [Server thread/INFO]: Modules -> 58 activés 8 désactivés: deathMessages, chatBubble, tablist, ranks, playerCombat, votifier, headDrop, jail
[16:55:59] [Server thread/INFO]: --------------------------------------------------
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by RyiaJones** [comment]
Having the same issues, Homes, Nicknames and loss of currency on a server I am currently building for a community. Everything is up to date.

**Comment by CloudeLecaw** [comment]
the issue is fixed with cmi 9.7.5.7 and cmilib 1.5.1.3

sadly if you got no Backup of your mysql data cmi table then the old data is lost.

but pls Update as fast as you can.

**Comment by Zrips** [comment]
Resolved with 9.7.5.7 version
